### PR TITLE
 gr-uhd: fix directional tune commands being dropped when TX and RX are the same frequency

### PR DESCRIPTION
**Pull request by jacobagilbert**
This is a [long overdue] fix for #1814 which adds a class member bool which will force a tune to occur if the 'direction' key is provided, and the value of that key is a symbol.

This PR also includes a commit fixing some misspellings in gr-uhd.

This should be backported all the way back to 3.7 in my opinion, but first I would like to see if this is acceptable.

### CODE REVIEW

File: gr-uhd/apps/uhd_app.py [hunk 1]
```diff
@@ -325,7 +325,7 @@ def set_freq(self, freq, skip_sync=False):
                 treq = uhd.tune_request(
                     target_freq=freq,
                     rf_freq=freq,
-                    rf_freq_policy=uhd.tune_reqest.POLICY_MANUAL,
```
Comment (mormj) [inline gr-uhd/apps/uhd_app.py:328]: how did this ever work?

Comment (mrjacobagilbert) [reply gr-uhd/apps/uhd_app.py:328]: I think this is just an example of some code nobody ever uses. The variable `tune_reqest` does not appear within the UHD code base, and it was similarly misspelled in a couple other places (local variables that were consistently misspelled thus not causing any real problems) that this PR fixes.

**Comment by mrjacobagilbert** [comment]
I fixed the clang formatting failure, and added a reset to the force tune variable after processing of the command message.

**Review by mbr0wn — APPROVED** [review]
@jacobagilbert Sorry for the long overdue review. This is absolutely right.

**Pull request merged**